string zone
---
privacy_zones/ZoneControl[] controls